test/goldens/build_file_web_application

```diff
@@ -28,3 +31,22 @@ dart_web_application(
         ":silly_monkey",
     ],
 )
+dev_server(
+    name = "main_web_dartium_serve",
+    deps = _PUB_DEPS + [
+        ":silly_monkey",
+    ],
+    data = glob(["web/**"]),
+)
+dart_ddc_bundle(
+    name = "main_web_ddc_bundle",
+    entry_library = "web/main.dart",
```

Comment:
or actually for the deps you could just be safe and include all the same deps as `:silly_monkey` plus `:silly_monkey`...